modules/playlists/src/main/java/org/opencastproject/playlists/PlaylistEntry.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Licensed to The Apereo Foundation under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ *
+ * The Apereo Foundation licenses this file to you under the Educational
+ * Community License, Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License. You may obtain a copy of the License
+ * at:
+ *
+ *   http://opensource.org/licenses/ecl2.txt
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ *
+ */
+package org.opencastproject.playlists;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+/**
+ * A playlist entry belongs to a playlist and cannot exist without it. It holds a reference to some content (usually an
+ * Opencast event), as well as additional metadata.
+ */
+@Entity(name = "PlaylistEntry")
+@Table(name = "oc_playlist_entry")
+public class PlaylistEntry {
+
+  @Id
+  @GeneratedValue
+  @Column(name = "id")
+  private long id;
+
+  @Column(name = "content_id")
+  private String contentId;
+
+  private PlaylistEntryType type;
```

Comment:
It is stored in the DB, I just missed adding the annotation there (the `@Column` is implicit if the class is properly annotated, so this works properly without it. I just add `@Column` everywhere because I like to be explicit.)